docs/fsharp/tour.md

```diff
@@ -1,4 +1,158 @@
 # Tour of F# #
 
+The best way to learn about F# is to read and write F# code.  This tour of some of the features of the language will highlight some things and give you some code snippets that you can execute on your machine.  To learn about setting up a development environment, check out [Getting Started](tutorials/getting-started/index.md).
+
+There are two primary concepts in F#: functions and types.  This tour will emphasize features of the language which fall into these two concepts.
+
+## How to Run the Code Samples
+
+>![NOTE]
+Eventually, this documentation site will have the ability to run these code samples directly in your browser.
```

Comment:
Good call.  I've avoided linking to it because the site has been broken for folks, but given the specific thing being discussed here, it makes sense.